tests/store/artifact/utils/test_model_utils.py

```diff
@@ -10,17 +10,18 @@
 @pytest.mark.parametrize(
     ("uri", "expected_name", "expected_version"),
     [
-        ("models:/AdsModel1/0", "AdsModel1", 0),
-        ("models:/Ads Model 1/12345", "Ads Model 1", 12345),
-        ("models:/12345/67890", "12345", 67890),
-        ("models://profile@databricks/12345/67890", "12345", 67890),
+        ("models:/AdsModel1/0", "AdsModel1", "0"),
```

Comment:
Just double checking here, but is the casting to string backwards compatible (i.e., the namedtuple that's been created; does it need to be Optional[str] or could it be Optional[int])?

Reply:
It could be Optional[int] but then we'd need to cast it back to string and it felt like an unnecessary conversion back and forth